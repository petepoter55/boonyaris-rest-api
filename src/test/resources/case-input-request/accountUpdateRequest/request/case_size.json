{
  "id" : 2,
  "username": "123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890",
  "role": "12345678901",
  "firstname": "123456789012345678901234567890123456789012345678901234567890",
  "lastname": "123456789012345678901234567890123456789012345678901234567890",
  "email": "123456789012345678901234567890123456789012345678901234567890",
  "createBy": "usperadmin",
  "delFlag": null
}