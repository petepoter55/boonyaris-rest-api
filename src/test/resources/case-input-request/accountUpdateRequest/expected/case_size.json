[
  "username: size must be between 0 and 255",
  "role: size must be between 0 and 10",
  "firstname: size must be between 0 and 50",
  "lastname: size must be between 0 and 50",
  "email: size must be between 0 and 50"
]